File: testing/runtime-errors/visual-baic/runtime_error_1.vb
' Raspberry Pi Foundation
' Usage licensed under CC BY-SA 4
' Note: This file is designed to be copied out and compiled on your machine.
' To run this file you need to:
' 1. Copy the contents
' 2. Paste them into the Visual Basic IDE of your choice (Visual Studio, for example)
' 3. Compile the program
' 4. Run the program

Module Program

    ' The Main subroutine is the default entry point for a VB programModule Program
    Sub Main()
        Console.WriteLine("Please enter a number:")
        Dim number As Integer = Int32.Parse(Console.ReadLine())
        Console.WriteLine(number)
    End Sub

End Module

File: testing/runtime-errors/c-sharp/runtime-error1.cs
/*
Raspberry Pi Foundation
Developed to be used alongside Isaac Computer Science, part of the National Centre for Computing Education
Usage licensed under CC BY-SA 4

Note: This file is designed to be copied out and compiled on your machine.
In order for it to compile properly you need to ensure that the project name is the same as the
"namespace" in this file.To run this file you need to:
1. Copy the contents
2. Paste them into the C# IDE of your choice (Microsoft Visual Studio, for example)
3. Change the namespace to match your project (if necessary)
4. Compile the program
5. Run the program
*/

using System;

namespace IsaacCodeSamples
{
    class Testing
    {
        // The Main method is the entry point for all C# programs
        public static void Main() {
            Console.WriteLine("Please enter a number:");
            int number = Int32.Parse(Console.ReadLine());
            Console.WriteLine(number);
        }
    }
}